portfolio/src/main/webapp/index.html

```diff
@@ -47,20 +47,83 @@
+      <div class="overlay"></div>
+      <div class="container description">
         <h3 class="text-center"> HELLO, MY NAME IS </h3>
         <h1 class="text-center"> INDHU GUNDA </h1>
       </div>
     </header>
 
-    <div id="content">
-      <h1>About Me!</h1>
-      <p>This is my portfolio.</p>
-      <img src="images/profile.jpg"/>
-      <p>Click here to get a random fact about me:</p>
-      <button onclick="addRandomFact()">Random Fact!</button>
-      <div id="fact-container"></div>
-      <p>Hi, I'm Indhu Gunda! I'm an incoming junior in Electrical and Computer Engineering at the University of Texas at Austin. Some of my hobbies include reading, handlettering, and watching TV. My all time favorite TV show is White Collar.</p>
+    <!-- about section -->
+    <div class="about" id="about">
+      <div class="container">
+        <h1 class="text-center">About Me</h1>
+        <div class="row">
+          <div class="col-lg-4 col-md-4 col-sm-12">
+            <img src="images/profile.jpg" class="img-fluid">
+          </div>
+          <div class="col-lg-8 col-md-8 col-sm-12 desc">
+            <h3>Indhu Gunda</h3>
+            <p>I'm an incoming junior in Electrical and Computer Engineering at the University of Texas at Austin. Some of my hobbies include reading, handlettering, and watching TV. My all time favorite TV show is White Collar.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- experience section -->
+    <div class="experience" id="experience">
+      <div class="overlay"></div>
+      <div class="container">
+        <h1 class="text-center">Experience</h1>
+        <div class="row card-deck">
+          <div class="col-md-4 col-1g-4 col-sm-12 card">
+            <div class="card-header">
+              <h6>Amazon</h6>
+            </div>
+            <div class="card-body">
+              <h4 class="card-title">AFE Software Development Intern</h4>
+              <ul>
+                <li>Built order issue resolution tool for the Supply Chain Optimization Technologies team that summarized order and shipment information from numerous sources and auto resolves issues in certain cases.</li>
+                <li>Designed product to help troubleshoot issues with orders/shipments all around the world.</li>
+              </ul>
+            </div>
+            <div class="card-footer">
+              <h6>Python and Java</h6>
+            </div>
+          </div>
+          <div class="col-md-4 col-1g-4 col-sm-12 card">
+            <div class="card-header">
+              <h6>Code.org</h6>
+            </div>
+            <div class="card-body">
+              <h4 class="card-title">Open Source Project</h4>
+              <ul>
+                <li>Creating a feature to facilitate access to pre-reader materials by enabling text to speech to auto play.</li>
+                <li>Building a customizable framework for teachers to employ this feature in their classrooms.</li>
```

Comment:
Nit: "Created" and "Built", everything else is past-tense

Reply:
fixed